Mage.Sets/src/mage/cards/r/RavenousSlime.java

```diff
@@ -112,16 +107,9 @@ public boolean checksEventType(GameEvent event, Game game) {
     @Override
     public boolean applies(GameEvent event, Ability source, Game game) {
         ZoneChangeEvent zEvent = (ZoneChangeEvent) event;
-        if (zEvent.getToZone() != Zone.GRAVEYARD) {
-            return false;
-        }
-        Permanent permanent = ((ZoneChangeEvent) event).getTarget();
-        if (permanent == null
-                || !game.getOpponents(source.getControllerId()).contains(permanent.getControllerId())) {
-            return false;
-        }
-        if (zEvent.getTarget() != null) { // if it comes from permanent, check if it was a creature on the battlefield
-            return zEvent.getTarget().isCreature();
-        } else return permanent.isCreature();
+        return zEvent.getTarget() != null
+                && zEvent.getTarget().isCreature()
+                && zEvent.getTarget().getControllerId() != source.getControllerId()
```

Comment:
There are range limits possible in games, so you must use opponent check, not equal command:
* like before: `game.getOpponents(source.getControllerId()).contains(permanent.getControllerId())`

Reply:
Gotcha. Updated the check to work with range of influence.

I copied the effect from Cry of the Carnarium, so that card will also need an update.